pkg/operation/botanist/component/logging/fluentoperator/fluent_operator_test.go

```diff
@@ -0,0 +1,1060 @@
+						ContainerPort: 2021,
+						Protocol:      "TCP",
+					},
+				},
+				Resources: corev1.ResourceRequirements{
+					Limits: corev1.ResourceList{
+						corev1.ResourceMemory: resource.MustParse("400Mi"),
+					},
+					Requests: corev1.ResourceList{
+						corev1.ResourceCPU:    resource.MustParse("150m"),
+						corev1.ResourceMemory: resource.MustParse("150Mi"),
+					},
+				},
+				ReadinessProbe: &corev1.Probe{
+					ProbeHandler: corev1.ProbeHandler{
+						HTTPGet: &corev1.HTTPGetAction{
+							Path: "/api/v1/metrics/prometheus",
+							Port: intstr.FromInt(2020),
+						},
+					},
+					PeriodSeconds: 10,
+				},
+				LivenessProbe: &corev1.Probe{
+					ProbeHandler: corev1.ProbeHandler{
+						HTTPGet: &corev1.HTTPGetAction{
+							Path: "/healthz",
+							Port: intstr.FromInt(2021),
+						},
+					},
+					PeriodSeconds:       300,
+					InitialDelaySeconds: 90,
+				},
+				Tolerations: []corev1.Toleration{
+					{
+						Key:    "node-role.kubernetes.io/master",
+						Effect: corev1.TaintEffectNoSchedule,
+					},
+					{
+						Key:    "node-role.kubernetes.io/control-plane",
+						Effect: corev1.TaintEffectNoSchedule,
+					},
+				},
+				Volumes: []corev1.Volume{
+					{
+						Name: "runlogjournal",
+						VolumeSource: corev1.VolumeSource{
+							HostPath: &corev1.HostPathVolumeSource{
+								Path: "/run/log/journal",
+							},
+						},
+					},
+					{
+						Name: "optfluent",
+						VolumeSource: corev1.VolumeSource{
+							HostPath: &corev1.HostPathVolumeSource{
+								Path: "/opt/fluentbit",
+							},
+						},
+					},
+					{
+						Name: "plugins",
+						VolumeSource: corev1.VolumeSource{
+							EmptyDir: &corev1.EmptyDirVolumeSource{},
+						},
+					},
+				},
+				VolumesMounts: []corev1.VolumeMount{
+					{
+						Name:      "runlogjournal",
+						MountPath: "/run/log/journal",
+					},
+					{
+						Name:      "optfluent",
+						MountPath: "/opt/fluentbit",
+					},
+					{
+						Name:      "plugins",
+						MountPath: "/fluent-bit/plugins",
+					},
+				},
+				InitContainers: []corev1.Container{
+					{
+						Name:  "install-plugin",
+						Image: image,
+						Command: []string{
+							"cp",
+							"/source/plugins/.",
+							"/plugins",
+						},
+						VolumeMounts: []corev1.VolumeMount{
+							{
+								Name:      "plugins",
+								MountPath: "/plugins",
+							},
+						},
+					},
+				},
+				RBACRules: []rbacv1.PolicyRule{
+					{
+						APIGroups: []string{"extensions.gardener.cloud"},
+						Resources: []string{"clusters"},
+						Verbs:     []string{"get", "list", "watch"},
+					},
+				},
+			},
+		}
+
+		clusterFluentBitConfig = &fluentresources.ClusterFluentBitConfig{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: fbName + "-config",
+				Labels: map[string]string{
+					"app.kubernetes.io/name": fbName,
+				},
+			},
+			Spec: fluentresources.FluentBitConfigSpec{
+				Service: &fluentresources.Service{
+					FlushSeconds: pointer.Int64Ptr(30),
```

Comment:
```suggestion
					FlushSeconds: pointer.Int64(30),
```

The same comment applies to all usages of the deprecated `pointer.{something}Ptr` funcs.